src/components/user-details/user-details.js

```diff
@@ -8,19 +8,19 @@ export class UserDetails extends Component {
         return (
             <div className='user-details col-md card'>
                 <img className='user-photo'
-                     src={this.props.avatar_url}
+                     src='#'
```

Comment:
Should we make a default values for each prop? This way you won't need to change it every time there are no data in store.